zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1149,7 +1170,18 @@ components:
         application/json:
           schema:
             $ref: "#/components/schemas/TopologyResponse"
-
+    ProcessInstanceSearchQueryResponse:
+      description: Process instance search response.
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ProcessInstanceSearchQueryResponse"
```

Comment:
🔧 This response is not used, as far as I can see. The respective defined path uses the schema directly, not this response. I have to admit, I am not sure if it's required to use such a response component or if using the schema directly is fine.